theme/static/style/color-palette-light.css

```diff
@@ -63,3 +63,21 @@ nav.primary a.active, #pagination a.active {
     background-color: #6e827b;
     color: #e3edea;
 }
+
+#mc_embed_signup {
```

Comment:
Please restore back the <link> tag which imports MailChimp styles in the head section as it was the first time around.

Instead of defining the colors again add this after the `.stripe-gray` selector and use the existing settings.